core/arch/arm/include/kernel/secure_partition.h

```diff
@@ -44,6 +44,8 @@ struct sp_session {
 	const void *fdt;
 	bool is_initialized;
 	TEE_UUID ffa_uuid;
+	uint32_t ns_interrupts_action;
```

Comment:
The names of these fields are too long.

Reply:
Ok, I'm changing them to something shorter.